src/modules/SX128x/SX128x.cpp

```diff
@@ -1064,6 +1112,39 @@ float SX128x::getSNR() {
   }
 }
 
+double SX128x::getFrequencyError() {
```

Comment:
`float` should be used instead of `double` - there's no need for double-precision, and platforms with single-precision FPU might not be able to take advatnage of it.

Reply:
OK changing method and calculations from double to float.  I used the Semtech frequency error example code from mbed and they used double for the calculation.  I just continued using double.  I also used setGainControl as an example of reading the SX128x registers. I am not sure if the  RADIOLIB_ASSERT(state) is required after each read?

The method has been tested on an ESP32 SPI connected to an Ebyte E28 2G4M27S transceiver based on the SX1280.  Using a second E28 to cycle through -200kHz of receiver frequency to +250kHz and the results of the Frequency Error method were reasonably accurate with the changing transmit frequency.

No performance testing was done other than receiving four packets a second during the frequency sweep test.